packages/lesswrong/lib/collections/comments/custom_fields.js

```diff
@@ -56,7 +56,7 @@ Comments.addField([
       viewableBy: ['guests'],
```

Comment:
Comment above this (GitHub won't let me comment on unchanged lines( should be updated to not say Ory Editor.